aw_transform/classify.py

```diff
@@ -15,7 +15,7 @@ class Rule:
     ignore_case: bool
 
     def __init__(self, rules: Dict[str, Any]):
-        self.select_keys = rules.get("select_keys", None)
+        self.select_keys = rules.get("select_keys", ["app", "title"])
```

Comment:
This would be better as a default when creating a new rule in the frontend, not made an implicit default in the transform imo.

Reply:
It doesn't look like there is an existing component we can use for a multiselect/array form input right now. Am I missing something?

Here's what I propose here:

* Merge this PR to avoid accidental categorization (matching against `url` could yield bad results)
* Having an explicit `all` magic key will be helpful regardless
* Work on getting a array input + `select_keys` available on the frontend

Reply:
>It doesn't look like there is an existing component we can use for a multiselect/array form input right now.

No, but should be easy to add. Could even just put a text field, and ask users to comma-separate.